mui/CHANGELOG.md

```diff
@@ -113,5 +113,5 @@ Previous versions listed after this indicator refer to our deprecated `@pxblue`
 
 ```tsx
 // new import syntax
-import IconName from '@pxblue/icons-mui/IconName';
+import IconName from '@brightlayer-ui/icons-mui/IconName';
```

Comment:
I wouldn't change any pxb refs in the older changelogs; these notes probably don't even apply to this package anymore.